### ISSUE TITLE
Location not supported even though websites support the UK ?

### ISSUE THREAD

**Issue by atv2016** [aex351/home-assistant-neerslag-app#38]
I am getting not in coverage or Not found: location must be inside the Netherlands or Belgium when installing these. I’m in the UK, but both sites support the UK.

What’s going on ?

**Comment by aex351** [comment]
It seems you are using GPS coordinates that is currently not supported by the API (Buienalarm/Buienradar) that is being used by the `neerslag-app`. It could well be that on their main website (Buienalarm/Buienradar)  they use a different API that does support your location. Unfortunately, the last time I checked, I did not have access to that. 

I am looking however into include others API's from different providers. That should extend the coverage. Which includes the UK.

**Comment by atv2016** [comment]
Yes it seems it works fine on their website. I thought you were using their API? And yes it would be great if you could support more forecasters. Thanks!
________________________________

**Comment by aex351** [comment]
Correct, I am using their API's. Only their API's seem to limited to the current supported countries. If there are API's available for other countries I can implement those.
